options.php

```diff
@@ -136,39 +136,39 @@ function wpmautic_script_location() {
 			<input
 				type="radio"
 				name="wpmautic_options[script_location]"
-				value="disabled"
```

Comment:
Why changing the option order here ?

Reply:
I think most users will have tracking enabled by default. And disabling it will be the least popular choice. Therefore, I moved it to the bottom.